src/utils.js

```diff
@@ -198,18 +198,24 @@ function htmlTreeAsString(elem) {
  */
 function htmlElementAsString(elem) {
     var out = [],
+        className,
         classes,
         key,
         attr,
         i;
 
+    if (!elem || !('tagName' in elem)) {
```

Comment:
Probably just overthinking. If `tagName` is falsy, we are in trouble, so yeah that check is fine. Changed.